src/services/geolocation.service.ts

```diff
@@ -25,6 +28,14 @@ const nearest = async (
     });
 };
 
+const buildGeolocationUrl = (postcode: string, pagination: PaginationOptions, filters: ResultsFilters): string => {
+  let url = `${process.env.GEOLOCATION_URL}/${postcode}?page=${pagination.page}&limit=${pagination.limit}`;
```

Comment:
Is it worth using the TypeScript inbuilt url method to create the url and the queries here or can you pass the query parameters through to the `request.get`?